components/Modal/CreateImageUpload.js

```diff
@@ -442,88 +496,297 @@ class CreateImageUploadModal extends React.Component {
+                  </Button>
+                </Popover>
+              </div>
+              <TextInput
+                className="pf-c-form-control"
+                value={this.state.uploadSettings["region"]}
+                type="text"
+                id="region-input"
+                name="region"
+                onChange={this.setUploadSettings}
+              />
+            </div>
+          </Form>
         </React.Fragment>
       )
-    });
+    };
 
-    const uploadStep = (provider, displayName) => ({
-      name: `Upload to ${displayName}`,
-      steps: [uploadAuth(provider), uploadSettings(provider)]
-    });
+    const uploadStep = {
+      name: `Upload to AWS`,
+      steps: [awsUploadAuth, awsUploadSettings]
+    };
+
+    const awsReviewStep = uploadService === "aws" && (
+      <TextContent>
+        <div className="pf-l-flex pf-u-display-flex">
+          <h3 className="pf-l-flex__item pf-u-mt-2xl pf-u-mb-md">
+            <FormattedMessage defaultMessage="Upload to Amazon" />
+          </h3>
+          <Popover
+            className="pf-l-flex__item"
+            id="aws-review-popover"
+            bodyContent={
+              <TextContent>
+                <p>
+                  <FormattedMessage
+                    defaultMessage="
+                      Image Builder can upload images you create to an {bucket} in AWS and then import them into EC2. When the image build is complete
+                      and the upload action is successful, the image file is available in the AMI section of EC2. Most of the values required to upload
+                      the image can be found in the {console}.
+                    "
+                    values={{
+                      bucket: "S3 bucket",
+                      console: <a href="https://console.aws.amazon.com/console/home">AWS Management Console</a>
+                    }}
+                  />
+                </p>
+                <p>
+                  <FormattedMessage
+                    defaultMessage="
+                      This upload process requires that you have an {iam} role named {vmimport} to ensure that the image can
+                      be imported from the S3 {bucket} into EC2. For more details, refer to the {role}.
+                    "
+                    values={{
+                      bucket: "bucket",
+                      vmimport: <code>vmimport</code>,
+                      iam: "Identity and Access Management (IAM)",
+                      role: (
+                        <a href="https://docs.aws.amazon.com/vm-import/latest/userguide/vmie_prereqs.html#vmimport-role">
+                          AWS Required Service Role
+                        </a>
+                      )
+                    }}
+                  />
+                </p>
+              </TextContent>
+            }
+            aria-label="aws help"
+          >
+            <Button variant="plain" aria-label="aws help">
```

Comment:
Similar to a previous comment, acronyms used in `aria-label` should be all caps.